### PR TITLE
feat(conf): make unified report configurable

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
## Description

Checkbox to include/exclude tables in Unified Report.

### Changes

* Add table with headings in text field and checkbox.
* Include/exclude by enable/disable checkbox.
* Edit headings from text field.

## How to test

* Upload a package.
* Go to conf page and un-check headings and submit.
* Download unified report. unchecked tables under headings shouldn't exists.
* Change heading name and see if the changes appear in report.

### CODE REVIEW

**Comment by shaheemazmalmmd** [comment]
Will re-base the branch once https://github.com/fossology/fossology/pull/1747 gets merged.

**Comment by GMishx** [comment]
Can you please increase the width of the table? It is very difficult to see everything available in the text boxes.

image

**Comment by GMishx** [comment]
Also, the following two header texts did not change with the text from DB:
1. Export Restrictions
2. Clearing Protocol Change Log

File: src/www/ui/core-schema.dat [hunk 1]
```diff
@@ -1896,6 +1896,10 @@
   $Schema["TABLE"]["report_info"]["ri_copyrightnotes"]["ADD"] = "ALTER TABLE \"report_info\" ADD COLUMN \"ri_copyrightnotes\" text DEFAULT 'NA'::text";
   $Schema["TABLE"]["report_info"]["ri_copyrightnotes"]["ALTER"] = "ALTER TABLE \"report_info\" ALTER COLUMN \"ri_copyrightnotes\" SET NOT NULL, ALTER COLUMN \"ri_copyrightnotes\" SET DEFAULT 'NA'::text";
 
+  $Schema["TABLE"]["report_info"]["ri_unifiedcolumns"]["DESC"] = "";
```
Comment (GMishx) [inline src/www/ui/core-schema.dat:1899]: It will be nice to have a column description.

File: src/unifiedreport/agent/unifiedreport.php [hunk 2]
```diff
@@ -706,113 +706,177 @@ private function writeReport($contents, $uploadId, $groupId, $userId)
...
+    }
+
+    $heading = array_keys($unifiedColumns['acknowledgements'])[0];
+    $isEnabled = array_values($unifiedColumns['acknowledgements'])[0];
+    if ($isEnabled) {
+      /* Display acknowledgement */
+      $titleSubHeadingAcknowledgement = "(Reference to the license, Text of acknowledgements, File path)";
+      $this->bulkLicenseTable($section, $heading, $contents['licenseAcknowledgements']['statements'], $titleSubHeadingAcknowledgement);
+    }
+    print_r($unifiedColumns['exportrestrictions']);
```
Comment (GMishx) [inline src/unifiedreport/agent/unifiedreport.php:738]: Please remove debugging statement.

**Review by GMishx — CHANGES_REQUESTED** [review]
One tiny change.

**Review by GMishx — APPROVED** [review]
Changes looks good. Tested, working fine.

**Pull request merged**